Vampirika - Legion balance and arthas ult nerf

vampirika opened the pull request:
Legion:

- Improved burning archer damage for both attacks, normalised values. (overall dps/value is comparative to meat wagon)
  - Attack 1 damage range increased 13-16 -> 18-22.
  - Attack 1 attack range increased from 550 -> 600. (same as attack 2)
  - Attack 2 damage range increased 10-14 -> 18-22.
  - Attack 2 attack cooldown decreased 1.5 -> 1.8. (same as attack 1)
- Infiltrator better value.
  - Gold cost decreased 30 -> 25.
  - Attack cooldown decreased 1.9 -> 1.5.
  - Damage base increased 10 -> 20.
- Rematerialisation chance increased 15% -> 20%.
- Barrack cost decreased 16 -> 12; Magic building cost decreased 15 -> 11; Altar cost increased 18 -> 24.
- Dreadlord astrall recall min/max time decreased 3/30s -> 2/20s.
- Legion heroes increase stat scaling and buffed abilities.
  - Mal'ganis stats increased; str/level 2.0 -> 2.4; int/level 3.25 -> 3.4.
  - Mal'ganis ultimate (doom) number of summons increased 2/4/6 -> 3/6/9; DoT damage increase 40/70/100 dps -> 50/90/130 dps.
  - Tichondrius stats increased; str/level 2.5->3.0.
  - Tichondrius vampiric aura value increased 10/15/20/25% -> 12/20/28/36%.
  - Anetheron stats decreased; str/level 4.0 -> 3.5; int/level 3.5-> 3.2.

Lordaeron:

- Reduced values of arthas ult 40/60/80 -> 30/50/70.


General:

- Gilneas has been put on a leash, and can now be called "dog" or "dogs" in commands.
- Bilgewater has found there is now more than a single goblin, and can now be called "goblins" in commands.
- Gnomes can now be properly liberated, literally (_typo fix_).

YakaryBovine commented on mapdata/WarcraftLegacies/AbilityData/Skin/1177628737.json:
```diff
@@ -75,7 +75,7 @@
       "Id": 1953854049,
       "Type": 3,
-      "Value": "Targets an area for solar bombardment, carpeting it with 28 bolts over the course of 14 seconds. Each bolt deals damage to enemy units and heals non-Undead allies for 150% of that amount. Undead enemies take 10% additional damage.  |n|n|cffffcc00Level 1|r - Each bolt deals 40 damage.|n|cffffcc00Level 2|r - Each bolt deals 60 damage. |n|cffffcc00Level 3|r - Each bolt deals 80 damag ...
+      "Value": "Targets an area for solar bombardment, carpeting it with 56 bolts over the course of 14 seconds. Each bolt deals damage to enemy units and heals non-Undead allies for 150% of that amount. Undead enemies take 10% additional damage.  |n|n|cffffcc00Level 1|r - Each bolt deals 30 damage.|n|cffffcc00Level 2|r - Each bolt deals 50 damage. |n|cffffcc00Level 3|r - Each bolt deals 70 damag ...
```
Oops! Nice correction on the math

YakaryBovine replied:
Need to update the per-level version of the tooltip as well, currently still has the old values.

YakaryBovine reviewed (changes requested):
Only 1 thing to fix, otherwise perfect

YakaryBovine reviewed (approved):
Looks great

vampirika commented:
Druids:

Reduced dryad upgrade costs.
100g -> 50g
Buffed dryad poison upgrades.
Crippling poison attack speed slow up to 40% (previously 35%, was 40% after 2nd upgrade)
Deadly poison upgrade increased dps 5->6 but reduced duration 5->4; total damage of DoT 20->24.
Warsong:

Fixed grom stats after bloodpact matching regular form. (oopsie!)
Gilneas:

Reduced tess W attacked speed buff 200/300/400/500% -> 175/250/325/400%.
Fixed a bug with tess shooting too many projectiles at low levels.
Scourge:

Increased efficacy of anub'arak ultimate
Number of locusts from 20/30/40 -> 25/35/45.
Locust attack cooldown down 1.0 -> 0.5.
Reduced heal threshold from 20/30/40 to 20 at all levels. (Locusts will return to anub'arak sooner).
Locust unleash instantly.
Increased strength of beetles at higher levels
Beetles level 2 health 550 -> 650.
Beetles level 3 health 650 -> 850.
Beetles level 4 uses level 3 beetles (as before) but now spawns 4 instead of 2.
Beetle manacost lowered to 30 at all levels.
Lordaeron:

Removed West goldmine for lordaeron, increased starting goldmine by 200g. (-400g overall).
Removed some creeps associated with lordaeron creeping, particularly in scholomance.
